tests/test_app_metadata.py

```diff
@@ -69,6 +69,18 @@
                 "libc": "glibc",
             },
         ),
+        (
+            True,
+            "nodejs",
```

Comment:
The runtime name in gprofiler is `nodejs`, but it will search for the `nodejs` name in the stack, while the name is actually `node`. You can override `application_executable` for node as well